source/macos/AdaptiveCards/AdaptiveCards/Renderers/TextBlockRenderer.swift

```diff
@@ -56,6 +55,17 @@ class TextBlockRenderer: NSObject, BaseCardElementRendererProtocol {
             // Hide accessibility Element
         }
         
+        // Set compression priority higher value means that we don’t want the view to shrink smaller than the intrinsic content size.
+        textView.setContentCompressionResistancePriority(.required, for: .vertical)
```

Comment:
Can you just check if this doesn't mess with max lines property of text block?

Reply:
Yes, it won’t affect..
<img width="362" alt="Screenshot 2022-08-01 at 9 39 24 AM" src="https://user-images.githubusercontent.com/105660080/182070524-e6283e8f-06a3-42bd-89c3-7e954fa13aa0.png">